product_template_tags/views/product_template_tag.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="product_template_tag_form_view">
```

Comment:
just aesthetic change:
id before model

id pattern:
For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...